src/utils/mutations.ts

```diff
@@ -516,6 +624,34 @@ export async function denyTask(classID: string, studentID: string, taskID: strin
 	}
 }
 
+export async function confirmAllTasks(tasks: CompletedTask[], classID: string) {
```

Comment:
If you have time could you factor the code from confirmTask and confirmAllTasks so that there's no repetition? You could probably abstract the common parts into a non-exported helper function to do the refactoring without changing the actual function declarations